spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestMigrateTableProcedure.java

```diff
@@ -160,30 +154,32 @@ public void testMigrateWithInvalidMetricsConfig() throws IOException {
         .hasMessageStartingWith("Invalid metrics config");
   }
 
-  @Test
+  @TestTemplate
   public void testMigrateWithConflictingProps() throws IOException {
-    Assume.assumeTrue(catalogName.equals("spark_catalog"));
+    assumeThat(catalogName).isEqualToIgnoringCase("spark_catalog");
 
-    String location = temp.newFolder().toString();
+    String location = temp.toFile().toString();
     sql(
         "CREATE TABLE %s (id bigint NOT NULL, data string) USING parquet LOCATION '%s'",
         tableName, location);
     sql("INSERT INTO TABLE %s VALUES (1, 'a')", tableName);
 
     Object result =
         scalarSql("CALL %s.system.migrate('%s', map('migrated', 'false'))", catalogName, tableName);
-    Assert.assertEquals("Should have added one file", 1L, result);
+    assertThat(result).as("Should have added one file").isEqualTo(1L);
 
     assertEquals(
         "Should have expected rows",
         ImmutableList.of(row(1L, "a")),
         sql("SELECT * FROM %s", tableName));
 
     Table table = validationCatalog.loadTable(tableIdent);
-    Assert.assertEquals("Should override user value", "true", table.properties().get("migrated"));
+    assertThat(table.properties().get("migrated"))
```

Comment:
should also be using `containsEntry()`as this will show the full output of the map if the assertion ever fails